common/main/cpp/native_c4listener.cc

```diff
@@ -394,6 +394,8 @@ static C4Listener *startListener(
     config.directory = path;
     config.allowCreateDBs = allowCreateDBs;
     config.allowDeleteDBs = allowDeleteDBs;
+    config.allowCreateCollections = false;
+    config.allowDeleteCollections = false;
```

Comment:
Since this was just a smoke test, it is not needed for the final code.  These two will already be false by virtue of you creating the struct via `{}`.